packages/dropbear/build.sh

```diff
@@ -9,6 +9,9 @@ TERMUX_PKG_SHA256=48235d10b37775dbda59341ac0c4b239b82ad6318c31568b985730c788aac5
 TERMUX_PKG_DEPENDS="termux-auth, zlib"
 TERMUX_PKG_CONFLICTS="openssh"
 TERMUX_PKG_BUILD_IN_SRC=true
+if [ $TERMUX_PACKAGE_FORMAT = "pacman" ]; then
+	TERMUX_PKG_PROVIDES="openssh"
```

Comment:
It doesn't provide OpenSSH but has symlinks `scp` and `ssh` which ARE NOT compatible with OpenSSH.

Reply:
which is better to specify `scp` or `ssh`?